pkg/minikube/constants/constants.go

```diff
@@ -58,10 +62,12 @@ func MakeMiniPath(fileName ...string) string {
 // getMinishiftHomeDir determines the Minishift home directory where all state information is kept.
 // The default directory is .minishift in the users HOME directory which can be overwritten by the MINISHIFT_HOME
 // environment variable
-func getMinishiftHomeDir() string {
+func GetMinishiftHomeDir() string {
 	homeEnv, ok := os.LookupEnv(MiniShiftHomeEnv)
 	if ok {
 		return homeEnv
+	} else if ProfileName != DefaultProfileName {
```

Comment:
-1, we will need to differentiate between MINISHIFT_HOME and the home for a given profile/instance.